package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sprintly-ui",
   "title": "Sprintly UI",
   "description": "A React-based UI kit for building Sprintly interfaces",
-  "version": "0.3.2",
+  "version": "0.4.2",
```

Comment:
This is technically a backwards incompatible change, as people who had this before will now see broken styles.